01.w_Defects/uninit_var.c

```diff
@@ -116,7 +116,7 @@ void uninit_var_007 ()
  */
 int uninit_var_008_func_001 (int a)
 {
-	static int ret;
+	static int ret = 2;
```

Comment:
static vars are initialized to zero, but if it starts with the value zero then the line with the error never gets called.